doc/schemas/gov_openapi.json

```diff
@@ -1283,7 +1325,7 @@
   "info": {
     "description": "This API is used to submit and query proposals which affect CCF's public governance tables.",
     "title": "CCF Governance API",
-    "version": "4.1.3"
+    "version": "4.1.4"
```

Comment:
Arguably 4.2.0, but this is a pretty obscure version number.

Reply:
The API here is only updated in response fields - we're not asking users to submit new bodies or new requests, just reporting a new field on the node config of some responses. So I think a minor minor patch version is sufficient.

Reply:
Sure, but it does seem like a new feature, which results in new behaviour if opted into. Anyway, pretty moot, nobody looks at this number.